api-gateway/src/main/resources/static/scripts/vet-details/vet-details.template.html

```diff
@@ -117,6 +117,20 @@ <h5 class="mb-3">Work Day</h5>
                     </div>
                 </div>
 
+                <div class="row g-3">
+                    <div class="col-sm-12">
+                        <h5 class="form-label">Ratings</h5>
+                        <div ng-if="$ctrl.ratings.length === 0">
+                            This vet has no ratings.
+                        </div>
+                        <div ng-repeat="rating in $ctrl.ratings" ng-if="$ctrl.ratings.length > 0">
+                            <div class="rating-item">
+                                <span class="rating-value">{{ rating.rateScore }} / 5</span>
```

Comment:
Hey, 
{{rating.rateScore}} is the "4" and the "3" from the two ratings. The /5 is displayed on the UI, after the "Edit Vet" Button, since the score rate is always out of 5. That is why the span containt "/5" 
![image](https://github.com/cgerard321/champlain_petclinic/assets/46633364/2de8ad1e-9b7e-4c8f-b108-4b0c7449ace5)

Reply:
Oh my bad, I couldn't see at first.